routes/cds-services.js

```diff
@@ -213,7 +214,9 @@ async function call(req, res, next) {
       return;
     }
   }
+  console.timeEnd('CDS Prefetch');
 
+  console.time('Custome API call and data translate')
```

Comment:
Typo: `Custome` -> `Custom`. Here and on line 254 where this timer is ended.

Reply:
Resolved by commit https://github.com/ccsm-cds-tools/ccsm-cds-service/pull/27/commits/ee118bf3711337af839b498178441669a6bf9784.